### ISSUE TITLE
Errors when running in UCP

### ISSUE THREAD

**Issue by alanwill** [nicolaka/voteapp-base#1]
I'm getting these errors when running this against UCP:

```
result-app_1 | Error performing query: error: relation "votes" does not exist
db           | ERROR:  relation "votes" does not exist at character 38
db           | STATEMENT:  SELECT vote, COUNT(id) AS count FROM votes GROUP BY vote
worker_1     | Failed to connect to redis - retrying
```

Any idea why?

**Comment by nicolaka** [comment]
@alanwill can you double check ( by execing into a container) that you can reach all other containers by name? i've had this problem before and for some reason it was due to overlay networking not resolving the container name.

**Comment by alanwill** [comment]
@nicolaka yes that's the same problem I'm having. The overlay network is on all the hosts, but I can't ping redis and from that container I can't ping the others, like db.

How did you fix the problem when you ran into it?

**Comment by alanwill** [comment]
I moved the redis container to another host, the same on as db and it's reachable now but I'm still getting the errors from db

```
db           | ERROR:  relation "votes" does not exist at character 38
db           | STATEMENT:  SELECT vote, COUNT(id) AS count FROM votes GROUP BY vote
```

Have you seen those before?

**Comment by nicolaka** [comment]
are you sure you opened up all required mh networking ports ? (https://docs.docker.com/ucp/plan-production-install/#ports-used). if you can run 2 test containers (C1,C2) on the same overlay and try to ping the and see if the problem persists?

**Comment by alanwill** [comment]
Yes those ports are open and all the other containers which are on different hosts can all speak to each other. I was only able to fix the redis ping issue by putting it on the same host but even after doing that the db container is complaining for what looks like a data issue in postgres itself.

**Comment by nicolaka** [comment]
seems like overlays are not working. Is that the case if you test overlay separately?
